samples/subsys/ipc/openamp/sysbuild.cmake

```diff
@@ -14,6 +14,7 @@ ExternalZephyrProject_Add(
 # remote core's build, such as the output image's LMA
 add_dependencies(${DEFAULT_IMAGE} openamp_remote)
 sysbuild_add_dependencies(CONFIGURE ${DEFAULT_IMAGE} openamp_remote)
+sysbuild_add_dependencies(FLASH ${DEFAULT_IMAGE} openamp_remote)
```

Comment:
If there's no need for debuging, is this sample working on M4 and M7 from i.MXRT1170?
In the commit message you state that this change adds the M4 code section in the sysbuild image - so how was this working before?

Reply:
If there is not need for debugging, then a west flash will load the entire image and the sample will work correctly. I don't exactly know where the branching path occurs, but without this line of code the west debug command requires that the image was already loaded on the board prior.